whitelist.py

```diff
@@ -20,6 +20,7 @@
 patch_new_user_classes  # unused variable (monkey/tests/infection_monkey/utils/test_auto_new_user_factory.py:25)
 patch_new_user_classes  # unused variable (monkey/tests/infection_monkey/utils/test_auto_new_user_factory.py:31)
 mock_home_env  # unused variable (monkey/tests/monkey_island/cc/server_utils/test_island_logger.py:20)
+custom_pba_directory  # unused variable (monkey/tests/monkey_island/cc/services/test_post_breach_files.py:20)
```

Comment:
Any reason for adding this here exactly? If not, maybe we should make it a rule to add these at the end of the file along with the PR/commit it was introduced in so it's easy to track, if ever needed.

Reply:
I put it here because it's among the other `monkey/tests/monkey_island` files. But we can just make a rule of adding whitelist lines at the end in the future.

Reply:
I don't understand this. I don't see any `custom_pba_directory` in `monkey/tests/monkey_island/cc/services/test_post_breach_files.py` on line 20. I presume it's talking about the fixture, right?

Reply:
Yes, a lot of the line numbers in this file probably don't match. Maybe we should remove them, the filename should be enough.